src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx

```diff
@@ -367,6 +369,11 @@ int main_(int argc, const char** argv) {
       inputFiles.push_back(psd.getFile());
     }
 
+    for (auto section : sectionsToAddReplace) {
```

Comment:
auto& (or const auto&) ?

Reply:
Issue fixed with `const auto &` here and other parts of the code base that has a similar pattern.